exercises/sampler-built-in-types/test.ml

```diff
@@ -0,0 +1,51 @@
+open Test_lib
+open Report
+
+(*A first way to grade this function with randomly generated tests 
+  is by using the default sampler.*)
+let exercise_1 =
+  Section ([ Text "Function: "; Code "identity" ],
+           test_function_1_against_solution
+             [%ty: int -> int] "identity"
+             ~sampler:(fun () -> Random.int 32)
```

Comment:
The comment seems contradictory as you do not use the "default" value of `~sampler` but define it manually instead. Please rephrase.